### PR TITLE
Add smoke tests for plug-in system

### PR DESCRIPTION
**Pull request by evidolob**
This pull provide two test for plug-in system, one for frontend another for backend plugin.
Also it provide fix for plugin view, add missing blue line when plugin tab is active, and proper refresh view when new plugin installed.

### CODE REVIEW

File: packages/plugin-ext/src/main/browser/plugin-ext-widget.tsx [hunk 1]
```diff
@@ -22,28 +22,36 @@ import { HostedPluginServer, PluginMetadata } from '../../common/plugin-protocol
...
         this.title.label = 'Plugins';
         this.title.caption = 'Plugins';
         this.title.iconClass = 'fa plugins-tab-icon';
         this.addClass('theia-plugins');
-
+        this.node.tabIndex = 0;
         this.update();
         this.fetchPlugins();
+
+        this.disposableListener = this.notificationService.event(() => {
```
Comment (akosyakov) [inline packages/plugin-ext/src/main/browser/plugin-ext-widget.tsx]: fyi: `this.toDisposeOnDetach.push(...)`?

**Comment by benoitf** [comment]
it seems CI is failing

**Comment by benoitf** [comment]
```
@theia/example-browser: [chrome #0-3] 1) theia backend plugins should load and initialize frontend plugin:
@theia/example-browser: [chrome #0-3] element (".theia-Notification>.text") still not existing after 3000ms
```

**Comment by evidolob** [comment]
This pull is not relevant for now, looking on better way to write test

**Pull request closed without merging**